syntax = "proto3";

message XmlElement {
  string name = 1;
  repeated string attributes = 2;
  repeated XmlElement child_elements = 3;
  string text = 4;
}

message XmlDocument {
  XmlElement root = 1;
}

